templates/types/streaming/fastapi/app/api/routers/models.py

```diff
@@ -62,24 +92,21 @@ class ArtifactAnnotation(BaseModel):
 
 class Annotation(BaseModel):
     type: str
-    data: Union[AnnotationFileData, List[str], AgentAnnotation, ArtifactAnnotation]
-
-    def to_content(self) -> Optional[str]:
-        if self.type == "document_file":
-            if isinstance(self.data, AnnotationFileData):
-                # We only support generating context content for CSV files for now
-                csv_files = [file for file in self.data.files if file.filetype == "csv"]
-                if len(csv_files) > 0:
-                    return "Use data from following CSV raw content\n" + "\n".join(
```

Comment:
@marcusschiesser We no longer include the CSV content to the LLM; the file will be uploaded via the sandbox API. Therefore, the code interpreter tool will has issue when dealing with uploaded csv file because it don't have the file in the environment. We may consider deprecating the interpreter because the sandbox can do the same thing.